tutorials/plot_compute_covariance.py

```diff
@@ -35,26 +53,45 @@
 # useful if you use resting state as a noise baseline. Here we use the whole
 # empty room recording to compute the noise covariance (tmax=None is the same
 # as the end of the recording, see :func:`mne.compute_raw_covariance`).
-noise_cov = mne.compute_raw_covariance(raw_empty_room, tmin=0, tmax=None)
-
+#
+# Keep in mind that you want to match your empty room dataset to your
+# actual MEG data, processing-wise. Ensure that filters
+# are all the same and if you use ICA, apply it to your empty-room and subject
+# data equivalently. In this case we did not filter the data and
+# we don't use ICA. However, we do have bad channels and projections in
+# the MEG data, and, hence, we want to make sure they get stored in the
+# covariance object.
+
+raw_empty_room.info['bads'] = [
+    bb for bb in raw.info['bads'] if 'EEG' not in bb]
+raw_empty_room.add_proj(
+    [pp.copy() for pp in raw.info['projs'] if 'EEG' not in pp['desc']])
+
+raw_empty_room = mne.preprocessing.maxwell_filter(
+    raw_empty_room, cross_talk=ctc_fname, calibration=fine_cal_fname,
+    coord_frame='meg')
+
+noise_cov = mne.compute_raw_covariance(
+    raw_empty_room, tmin=0, tmax=None, method='diagonal_fixed')
```

Comment:
some explanation for `diagonal_fixed` choice would be good

Reply:
it was just debugging. I will remove it.